src/main/java/com/github/wovnio/wovnjava/Settings.java

```diff
@@ -129,6 +129,13 @@ private Lang verifyDefaultLang(String value) throws ConfigurationError {
         return lang;
     }
 
+    private String verifySnippetUrl(String value) throws ConfigurationError {
```

Comment:
this is an opt-in setting and we don't want to affect existing users. So default to the production url instead.

Reply:
use the `stringOrDefault` method. `verify` methods should be ones that can throw